src/serviceWorker.ts

```diff
@@ -11,9 +11,9 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
-  window.location.hostname === 'localhost' ||
+  window.location.hostname === "localhost" ||
```

Comment:
Ignore all this, seems I accidentally touched this file when looking for fetches and prettier got carried away.